### DISCUSSION TITLE
How can I use a precomputed contingency matrix ?

### DISCUSSION THREAD

**Discussion by Koowah** [MaxHalford/prince#162] [Q&A] [answered]
Hi all,

The title is pretty self-explanatory. Any help is welcome ! 😄 

Cheers

**Comment by MaxHalford** [comment]
Hey 👋 

This is regarding CA right?

**Reply by Koowah** [reply]
Thanks for the quick answer Max !
Man I can't believe I forgot to precise it haha apologies

This is regarding MCA

**Reply by MaxHalford** [reply, marked as answer]
Ok I release a new version (0.11) where there's a `one_hot` parameter available in MCA. If it's `False`, then you can pass a one-hot matrix. Let me know if something doesn't work.

**Reply by Koowah** [reply]
It works perfectly on my data. I'm gonna test it against other results and update the answer in case something is off.
Thank you very much your help 😄
